src/mlpack/bindings/python/print_class_defn.hpp

```diff
@@ -129,11 +129,14 @@ void PrintClassDefn(
   std::cout << std::endl;
   std::cout << "  def get_cpp_params(self, return_str=False):" << std::endl;
   std::cout << "    params = self._get_cpp_params()" << std::endl;
-  std::cout << "    return process_params_out(self, params, return_str=return_str)" << std::endl;
+  std::cout << "    return process_params_out(self, params, "
+    << "return_str=return_str)" << std::endl;
```

Comment:
```suggestion
      << "return_str=return_str)" << std::endl;
```
Should be a double indent on the wrapped line I suppose?